lang/src/lib.rs

```diff
@@ -59,6 +58,8 @@ pub use anchor_derive_accounts::Accounts;
 pub use borsh::{BorshDeserialize as AnchorDeserialize, BorshSerialize as AnchorSerialize};
 pub use solana_program;
 
+pub type AnchorResult<T> = Result<T, error::Error>;
```

Comment:
We should rename this to `Result<T>` as is done elsewhere in the Rust ecosystem, e.g., https://doc.rust-lang.org/std/io/type.Result.html. The `Anchor` part is implied when using the anchor crate.